actions.yaml

```diff
@@ -0,0 +1,7 @@
+cis-benchmark:
+  description: CIS Kubernetes Benchmark
+  params:
+    mode:
+      type: string
+      default: 'run'
```

Comment:
I think I like 'check' better than 'run'. Eventually we'll have two modes right, 'check' and 'apply'?

Reply:
Yes on an upcoming `apply` mode.  Can you live with `s/run/report`?  For some reason, I like `report` best.